docs/3.0.0-beta.x/community/contribute-with-docs.md

```diff
@@ -0,0 +1,486 @@
+
+Examples include the [Models Guide](/3.0.0-beta.x/guides/models.html#how-to-create-a-model) and the [Controllers Guide](/3.0.0-beta.x/guides/controllers.html).
+
+### Explanations
+
+**Explanations** cover why something was done and provide an understanding of the choices made through the course of the development of Strapi. These are generally written as **Articles** and found in the [Strapi Blog](https://blog.strapi.io).
+
+This _audience member_ is a developer who is already developing projects with Strapi or who is studying whether Strapi is a good fit for their project(s). **Explanations** are meant to explain different choices in the Roadmap, overall direction, and how certain features generally work.
+
+Examples include [Why we split the management of the Admin user and the Front-end Users](https://blog.strapi.io/why-we-split-the-management-of-the-admin-users-and-end-users/) and [Restricting Permissions in Strapi Admin](https://blog.strapi.io/admin-permissions/).
+
+These four categories follow the recommendations found in this article about [documentation by Divio](https://www.divio.com/blog/documentation/). We recommend reading it to gain a complete understanding of each type of content contained in our documentation.
+
+## The Workflow
+
+Please read the following sections before moving to create documentation. (Either new documentation or modifying existing documentation.)
+
+1. [Technical Requirements and the Installation of the Docs](/3.0.0-beta.x/community/contribute-with-docs.html#technical-requirements)
+2. [Understanding the Docs Structure](/3.0.0-beta.x/community/contribute-with-docs.html#understanding-the-docs-structure)
+3. [Submitting new documentation](/3.0.0-beta.x/community/contribute-with-docs.html#submitting-totally-new-documentation)
+4. [Adding/modifying/deleting documentation](/3.0.0-beta.x/community/contribute-with-docs.html#adding-modifying-deleting-content)
+
+- How to [submit an externally hosted tutorial](/3.0.0-beta.x/community/contribute-with-docs.html#submitting-an-externally-hosted-tutorial).
+- How to use the [Docs Style Guide](/3.0.0-beta.x/community/contribute-with-docs.html#docs-style-guide) to write better Docs.
+
+### Technical Requirements
+
+1. You have [Node v10.x.x](https://nodejs.org/en/download/) (LTS) or [Node v12.x.x](https://nodejs.org/en/download/current/) and [Yarn v1.2.0+](https://yarnpkg.com/en/).
+2. You are familiar with Git, and have it installed on your development environment.
+
+- Ensure you're using the [required versions of Node.js and npm](/3.0.0-beta.x/getting-started/install-requirements.html).
+- Fork the [Strapi repository](https://github.com/strapi/strapi) [to your own GitHub account](https://help.github.com/en/articles/fork-a-repo).
+- Clone your Strapi repository:
+
+`Path: ./Projects/`
+
+```bash
+git clone git@github.com:YOUR_USERNAME/strapi.git
+```
+
+- Install the dependencies **for the docs** section of the monorepo:
+
+`Path: ./Projects/`
```

Comment:
Are you sure that we really need this?

Reply:
Yarn install or npm install needs to be run in the docs/ directory in order for Vuepress to run. If someone goes directly to the run command, it will throw errors and I can't assume everyone will understand. 

So adding the dependencies is needed.

Reply:
If you're referring to the sentence about adding the dependencies. I think it makes things clear.

Reply:
I think he is referring to the Path especially we don't know what ./Projects refers to

Reply:
I added a note regarding this path. That should clarify this matter. @soupette @Aurelsicoko